### ISSUE TITLE
This does not work in Preact 10.0.0-beta.2

### ISSUE THREAD

**Issue by 38elements** [preactjs/enzyme-adapter-preact-pure#48]
I use `1.13.1`.
This works in `Preact 10.0.0-beta.1`.
This does not work in `Preact 10.0.0-beta.2`.

```
     TypeError: Cannot read property 'setState' of undefined
      at MountRenderer.render (node_modules/enzyme-adapter-preact-pure/build/src/MountRenderer.js:45:22)
      at new ReactWrapper (node_modules/enzyme/build/ReactWrapper.js:134:16)
      at mount (node_modules/enzyme/build/mount.js:21:10)
```

**Comment by robertknight** [comment]
Thank-you for the report. This has been resolved in v1.13.2.

**Comment by 38elements** [comment]
Thanks a lot.
This works normally.
